model/src/main/java/org/jboss/pnc/model/BuildConfiguration.java

```diff
@@ -60,6 +59,10 @@
     @NotNull
     private String name;
 
+    @Column(unique = true)
+    @NotNull
+    private GA ga;
```

Comment:
Is GA general enough to use it for all kind of builds? Also for non Java builds?